### PR TITLE
test: add optional env variable to support staging API scope format

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

**Review by nborde-CSM — APPROVED** [review]
LGTM :+1:

File: cypress/plugins/auth/authMSAL.config.js [hunk 1]
```diff
@@ -15,9 +15,11 @@ if (!CY_MSAL_CLIENT_SECRET)
   console.warn('MSAL client secret is empty ! Please set CY_MSAL_CLIENT_SECRET env variable when running cypress');
 
 const CY_MSAL_API_HOSTNAME_FOR_SCOPE = process.env.CY_MSAL_API_HOSTNAME_FOR_SCOPE;
-if (!CY_MSAL_API_HOSTNAME_FOR_SCOPE)
+const CY_MSAL_SCOPE = process.env.CY_MSAL_SCOPE;
+if (!CY_MSAL_API_HOSTNAME_FOR_SCOPE && !CY_MSAL_SCOPE)
   console.warn(
-    'API hostname for MSAL scope is empty ! Please set CY_MSAL_API_HOSTNAME_FOR_SCOPE env variable when running cypress'
+    'API hostname for MSAL scope is empty ! Please set CY_MSAL_SCOPE or CY_MSAL_API_HOSTNAME_FOR_SCOPE env variable ' +
```
Comment (esasova) [inline cypress/plugins/auth/authMSAL.config.js:21]: _empty !_ is french, no spaces needed in English :slightly_smiling_face:

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: cypress/plugins/auth/authMSAL.config.js (hunk 1 above)
Comment (csm-thu) [reply cypress/plugins/auth/authMSAL.config.js]: Fixed all occurrences in this file :slightly_smiling_face:

**Pull request merged**